daffodil-cli/src/it/scala/org/apache/daffodil/debugger/TestCLIDebugger.scala

```diff
@@ -1440,5 +1440,114 @@ class TestCLIdebugger {
     }
   }
 
+  @Test def test_CLI_Debugger_parse_unparser_not_available(): Unit = {
+    val schemaFile = Util.daffodilPath("daffodil-test/src/test/resources/org/apache/daffodil/section06/entities/charClassEntities.dfdl.xsd")
+    val inputFile = Util.daffodilPath("daffodil-cli/src/it/resources/org/apache/daffodil/CLI/input/input3.txt")
+    val (testSchemaFile, testInputFile) = if (Util.isWindows) (Util.cmdConvert(schemaFile), Util.cmdConvert(inputFile)) else (schemaFile, inputFile)
+
+    val shell = if (Util.isWindows) Util.start("", envp = DAFFODIL_JAVA_OPTS) else Util.start("")
+
+    try {
+      val cmd = String.format("%s -d parse -s %s %s", Util.binPath, testSchemaFile, testInputFile)
+      shell.sendLine(cmd)
+      shell.expect(contains("(debug)"))
+
+      shell.sendLine("break cell")
+      shell.expect(contains("1: cell"))
+      shell.sendLine("display info parser")
+      shell.sendLine("display info unparser")
+      shell.sendLine("continue")
+
+      shell.expect(contains("parser: <Element name='cell'><DelimiterStackParser>...</DelimiterStackParser></Element>"))
+      shell.expect(contains("unparser: not available"))
+
+      shell.sendLine("complete")
+
+      Util.expectExitCode(ExitCode.Success, shell)
+    } finally {
+      shell.close()
+    }
+  }
+
+  @Test def test_CLI_Debugger_parse_unparser_not_available_no_breaks(): Unit = {
```

Comment:
I'm not sure a separate test for no breaks is needed. I don't think this provides any real additional correctness. And integration tests are quite slow so we want to minimize unnecessary tests.

In fact, this "no breaks" test is probably the correct way to write the test, with the `expects` immediately after the associated `sendLine`. I would suggest removing the above test and just keeping this one (with `no_breaks` removed).

Same suggestion below for the parser not available tests.